server/controllers/mensa.ts

```diff
@@ -84,19 +85,75 @@ router.get('/', async (req, res, next) => {
       return result.sort((a, b) => moment(a.date).isBefore(moment(b.date)) ? -1 : 1).slice(0, amountOfReturnedDays);
     }, { ttl: CACHE_SECONDS });
 
+    return data;
+  } catch (error) {
+    console.log('Error');
+  }
+}
+
+router.get('/', async (req, res, next) => {
+  let mensaList:Mensa[] = [];
+  const mensaIDs = [130, 112, 200, 134, 132]; // list of ostfalia mensas
```

Comment:
Kann es sein, dass es fuer die Cafeteria in WF allgemein keine Plaene gibt? Sollte die trotzdem drin bleiben?